scripts/core/editor3/reducers/suggestions.jsx

```diff
@@ -535,4 +536,44 @@ const deleteCharacter = (editorState, style = null) => {
     return newState;
 };
 
+const deleteBlock = (editorState, blockKey) => {
```

Comment:
There's a function `removeBlock` in `/scripts/core/editor3/reducers/toolbar.jsx`. Could it be merged into one? It seems that your function does something different than just removes a block. I see that if the block is `null` is picks the block after and next selection depends on which block was picked. If it's not the same as `removeBlock` it'd be useful to give it a more descriptive name. Suggestions that contain a list are mentioned in the PR name, but nowhere in the code is there any reference to it. Could you add a comment before calling `deleteBlock` to explain why it's called from a function which should only delete a character?